CHANGELOG.unreleased.md

```diff
@@ -27,3 +29,6 @@
 > - major: if the change breaks compatibility
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
+
+- @xen-orchestra/backups minor
+- xo-server minor
```

Comment:
Shouldn't we release the proxy as well?

Reply:
Do we add the proxy entry to the CHANGELOG?

Reply:
You're right, that's what I said before, let's continue like that, we'll see in the future.